src/richie/apps/courses/models/course.py

```diff
@@ -771,6 +784,19 @@ class CourseRun(TranslatableModel):
         blank=False,
         max_length=20,
     )
+    offer = models.CharField(
+        _("offer"),
+        choices=CourseRunOffer.choices,
+        default=CourseRunOffer.FREE,
+        blank=False,
+        max_length=20,
+    )
+    price = models.FloatField(
+        _("price"),
+        default=0.0,
+        blank=False,
```

Comment:
As there is a default value we could allow it to be blanked nope ?

Reply:
Change done.

Reply:
Replaced the field with a DecimalField.